data/styles/clear/include/Basemap.mapcss

```diff
@@ -309,6 +320,10 @@ area|z12-13[landuse=field],
 area|z10-[leisure=nature_reserve],
 area|z10-[boundary=national_park],
 {fill-position: background; fill-opacity: 0.2; fill-color: @green6;}
+area|z10-[landuse=military][military=danger_area][wikipedia?]
+{fill-position: background; fill-opacity: 1.0; fill-color: @military;}
+area|z10-[landuse=military][military=cordon][wikipedia?]
+{fill-position: background; fill-opacity: 1.0; fill-color: @military;}
```

Comment:
To avoid copy-paste:
area|z10-[landuse=military][military=danger_area][wikipedia?],
area|z10-[landuse=military][military=cordon][wikipedia?],
{fill-position: background; fill-opacity: 1.0; fill-color: @military;}

Reply:
OK